src/test/ui/privacy/privacy5.stderr

```diff
@@ -225,7 +225,7 @@ LL |     let a = other::A(());
 LL | pub struct A(());
    |              -- a tuple struct constructor is private if any of its fields is private
```

Comment:
```suggestion
   |              -- constructor is private if any of the fields are private
```
We are saying "tuple struct constructor" in the primary message, so we can tidy up the label message now.
(For labels short is good.)
Also "its" -> "the" because the fields do not belong to the constructor.
Also, isn't "are" preferable to "is" here grammatically? There may be multiple private fields. Not sure.

Reply:
[English SE](https://ell.stackexchange.com/questions/34322/how-to-use-any-of) says it can work either way, while `is` being more formal.

Reply:
And should also keep the article 'a'.